packages/react/src/patterns/blocks/ContentGroupPictograms/__stories__/ContentGroupPictograms.stories.js

```diff
@@ -0,0 +1,119 @@
+
+    const pictogram1 = select(
+      'Element 1 pictogram (required)',
+      pictograms,
+      pictograms.Desktop
+    );
+
+    const pictogram2 = select(
+      'Element 2 pictogram (required)',
+      pictograms,
+      pictograms.Touch
+    );
+
+    const pictogram3 = select(
+      'Element 3 pictogram (required)',
+      pictograms,
+      pictograms.Pattern
+    );
+
+    /**
+     * Returns the react component based on the value in the pictogram variables
+     *
+     * @param {string} sel string that defines the returning pictogram
+     * @returns {*} JSX pictogram component
+     */
+    const selectPictogram = sel => {
+      switch (sel) {
+        case 'Desktop':
+          return Desktop;
+        case 'Pattern':
+          return Pattern;
+        case 'Touch':
+          return Touch;
+      }
+    };
+
+    const heading = text(
+      'Pattern title (required)',
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+    );
+
+    const items = [
+      {
+        heading: text(
+          'Element 1 Title (required)',
+          'Aliquam condimentum interdum'
+        ),
+        copy: text(
+          'Element 1 Copy (required)',
+          'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.'
+        ),
+        cta: object('Element 1 Link', {
+          type: 'text',
+          href: 'https://www.example.com',
+          copy: 'Lorem ipsum dolor',
+        }),
+        Pictogram: selectPictogram(pictogram1),
+      },
+      {
+        heading: text(
+          'Element 2 Title (required)',
+          'Aliquam condimentum interdum'
+        ),
+        copy: text(
+          'Element 2 Copy (required)',
+          'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.'
+        ),
+        cta: object('Element 2 Link', {
+          type: 'text',
+          href: 'https://www.example.com',
+          copy: 'Lorem ipsum dolor',
+        }),
+        Pictogram: selectPictogram(pictogram2),
+      },
+      {
+        heading: text(
+          'Element 3 Title (required)',
+          'Aliquam condimentum interdum'
+        ),
+        copy: text(
+          'Element 3 Copy (required)',
+          'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.'
+        ),
+        cta: object('Element 3 Link', {
+          type: 'text',
+          href: 'https://www.example.com',
+          copy: 'Lorem ipsum dolor',
+        }),
+        Pictogram: selectPictogram(pictogram3),
+      },
+    ];
+
+    return (
+      <div>
+        <ContentGroupPictograms heading={heading} items={items} />
+      </div>
```

Comment:
Could we put this inside of a grid so we can see it in context? I've been suggesting using something like the following since we include the full grid within storybook.

```jsx
<div class="bx--grid">
  <div class="bx--row">
    <div class="bx--col-sm-4 bx--col-lg-8 bx--offset-lg-4">
      <ContentGroupPictograms heading={heading} items={items} />
    </div>
  </div>
</div>
```